src/azimuth/view/string.c

```diff
@@ -404,14 +405,24 @@ void az_draw_paragraph(
           break;
         // For key name escapes, note for now which key_id we need to insert
         // the name of, and we'll handle it below.
-        case 'u': key_id = prefs->keys[AZ_PREFS_UP_KEY_INDEX]; break;
-        case 'd': key_id = prefs->keys[AZ_PREFS_DOWN_KEY_INDEX]; break;
-        case 'r': key_id = prefs->keys[AZ_PREFS_RIGHT_KEY_INDEX]; break;
-        case 'l': key_id = prefs->keys[AZ_PREFS_LEFT_KEY_INDEX]; break;
-        case 'f': key_id = prefs->keys[AZ_PREFS_FIRE_KEY_INDEX]; break;
-        case 'o': key_id = prefs->keys[AZ_PREFS_ORDN_KEY_INDEX]; break;
-        case 't': key_id = prefs->keys[AZ_PREFS_UTIL_KEY_INDEX]; break;
-        case 'p': key_id = prefs->keys[AZ_PREFS_PAUSE_KEY_INDEX]; break;
+        case 'u': key_id = key_for_control[AZ_CONTROL_UP]; break;
+        case 'd': key_id = key_for_control[AZ_CONTROL_DOWN]; break;
+        case 'r': key_id = key_for_control[AZ_CONTROL_RIGHT]; break;
+        case 'l': key_id = key_for_control[AZ_CONTROL_LEFT]; break;
+        case 'f': key_id = key_for_control[AZ_CONTROL_FIRE]; break;
+        case 'o': key_id = key_for_control[AZ_CONTROL_ORDN]; break;
+        case 't': key_id = key_for_control[AZ_CONTROL_UTIL]; break;
+        case 'p': key_id = key_for_control[AZ_CONTROL_PAUSE]; break;
+        case '0': key_id = key_for_control[AZ_CONTROL_BOMBS]; break;
```

Comment:
For completeness, could you also update the doc comment for `az_draw_paragraph()` in `src/azimuth/view/string.h` to mention the `$1` etc. escapes?

Reply:
of course, done.